### ISSUE TITLE
buggy overlapping text for multi-option <dd> entries

### ISSUE THREAD

**Issue by jzacsh** [Debian/debiman#143]
**tl;dr** synonymous flags become unreadable because they're accidentally rendered ontop of eacother's pixels.

It _appears_ that whenever there's an empty `dd` it causes the multiple synonymous `dt`s (eg: `-k` and `--DOS-names` synonyms or `zip(1)`) then the first n-1 synonyms have a necessarily empty `<dd>` that cause all the synomyms to get rendered on top of eachother and become illegible. Interestingly there's a seemingly relevant `clear: right` but it doesn't _appear_ to do anything, at least not now-a-days, so I'll send a fix to simply replace it.

### bug (before fix)

Here's a bunch of examples of the buggy behavior:
![image](https://user-images.githubusercontent.com/156228/171523500-7361acdd-e582-4448-a24a-948ae4c6e2b0.png)

### after fix
After fixing - by replacing [this particular `clear: right`](https://github.com/Debian/debiman/blob/680bd4844d7094884a103012396f230bb8ca4c98/assets/style.css#L544) with `clear: both` - here's what things look like:
![image](https://user-images.githubusercontent.com/156228/171523840-820d40ff-154c-4654-917c-d4695158edd1.png)

**Comment by stapelberg** [comment]
I’ve pushed your fix to the Debian-specific assets, too: https://salsa.debian.org/manpages-team/debian-assets/-/commit/9d7bae59c19070076032f7b8bce58c7d9e3e4a81

**Comment by stapelberg** [comment]
I have also started a full re-render so that manpages.debian.org picks up the changes. It’ll probably be a day or two until all pages are rendered and synchronized to all web servers.
